docs/sources/setup/install/helm/install-microservices/testing-deployment-with-kind.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Test the microservice Helm chart with Kind
+menuTitle: Test the microservice Helm chart with Kind
+description: Installing Loki in microservice (distributed) mode using the Helm chart.
+aliases:
+  - ../../../installation/helm/test-microservice/
+  - ../../../installation/helm/test-install-microservice/
+weight: 300
+keywords: 
+---
+
+
+# Test the microservice Helm chart with Kind
+<!-- vale Grafana.Quotes = NO -->
+<!-- vale Grafana.Quotes = YES -->
+
+This guide walks you through testing the Grafana Loki Helm chart in microservice mode using [Kind](https://kind.sigs.k8s.io/), a tool for running local Kubernetes clusters using Docker container nodes.
+
+## Prerequisites
+
+To run through this tutorial, you need the following tools installed:
```

Comment:
Not exactly sure if we need this topic, since we don't have a "testing" topic for the other installation methods.  I would assume that customers would test in a test environment before installing on a production environment.  Have you seen different behavior?

Reply:
So I have changed the name slightly to help. Essentially I wanted to provide our community with an easy to step by step guide on how to practise deploying this locally just so they can understand how the helm deploys without needing to set up a full cloud k8's stack. It mostly helped me as I was testing the new deployment helms that the team created :). But happy to remove if its overkill as might also be confusing

Reply:
LOL, just saw Trevor's comment and that you'd pulled it. That's OK too.